components/homme/test/reg_test/namelists/theta-fdc12-test21.nl

```diff
@@ -27,7 +27,7 @@
   dcmip2_x_xi       = 4000.0                    ! mountain wavelength   (m)
   limiter_option    = 9
   vert_remap_q_alg  = 1
-  hv_ref_profiles   = 1
+  hv_ref_profiles   = 6
```

Comment:
this namelist is used in the corresponding cxx test as well. i assume the new option is not in cxx code yet?

Reply:
I thought this test checked that cxx and F were BFB?

Reply:
thanks -- i got confused, the profiles are now only read from F.